### DISCUSSION TITLE
Price Consumer code

### DISCUSSION THREAD

**Discussion by kdadrian** [Cyfrin/foundry-full-course-cu#1466] [Q&A]
Hi everyone. I have been following along with the course and things have been going smoothly. However,in Lesson 4 where I have the the code on PriceConsumerV#.sol, the code from the documentation on chainlink is in Sepolia and not Kovan. Also, the deploy section of remix, I can't find injected Web3 in the environments

**Comment by EngrPips** [comment]
Hello @kdadrian , test nets are run out of good mind of people so from time to time the recommended test net to use for testing your codes and project do change. And i believe the recommended test net on the github repo of the course is currently sepolia as shown in the picture below, so i am not very sure what is confusing to you. 

<img width="1402" alt="Screenshot 2024-03-08 at 3 29 40 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/2a08d077-9fa7-44e6-b5c3-eca8eec095b5">

About not seeing injected web3 in the deploy section of remix , i must maybe you are not looking at the right tab. You should click on the tab where it says `Remix VM (Shanghai)` and you should see a drop down list, there you would see `Injected Provider` option , just like i have it in the picture below 

<img width="1440" alt="Screenshot 2024-03-08 at 3 32 06 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/04631da9-776c-42ee-985d-a6c90e9e478a">
